lib/utils/middlewares/azure_api_auth.ts

```diff
@@ -31,10 +31,10 @@ export enum UserGroup {
   // profiles: create and update full profile
   ApiProfileWrite = "ApiProfileWrite",
 
-  // organizations: read organization attributes
-  ApiOrganizationRead = "ApiOrganizationRead",
-  // organizations: create and update organizations
-  ApiOrganizationWrite = "ApiOrganizationWrite",
+  // services: read services attributes
+  ApiServiceRead = "ApiServiceRead",
+  // services: create and update services
+  ApiServiceWrite = "ApiServiceWrite",
```

Comment:
we must also rename the corresponding groups in the Azure API Management service

Reply:
ok

Reply:
done.